fdbrpc/include/fdbrpc/TenantInfo.h

```diff
@@ -33,8 +33,9 @@ struct TenantInfo {
 
 	Arena arena;
 	Optional<TenantNameRef> name;
```

Comment:
Q: If we eventually make tenant name no longer part of the internal API, will we end up removing this field and introduce a `Optional<TenantName>` that only client uses to send request?

Reply:
Yeah, there is a plan to remove the tenant name from the `TenantInfo` object, and the client will be responsible for converting the name to an ID before sending any requests.

Reply:
Great. Thanks!